lib/migrator.ex

```diff
@@ -232,7 +292,7 @@ defmodule EctoImmigrant.Migrator do
   end
 
   defp extract_module(file, _name) do
-    modules = Code.require_file(file)
+    modules = Code.compile_file(file)
```

Comment:
This change allow users to rerun the migration again when using a release version. When u only require a file, if you make any changes to file, this doesn't load the module again and creates a cache.

Reply:
Great work! Thanks for contributing.